wicket-core/src/main/java/org/apache/wicket/markup/MarkupFactory.java

```diff
@@ -61,6 +64,8 @@ public static MarkupFactory get()
 		return Application.get().getMarkupSettings().getMarkupFactory();
 	}
 
+	private final LinkedHashMap<IMarkupFilter, Class<? extends IMarkupFilter>> additionalMarkupFilters = new LinkedHashMap<>();

```

Comment:
IMO it would be better to change `private final LinkedHashMap` -> `private final Map`